superset-frontend/src/features/databases/CSVUploadModal/index.tsx

```diff
@@ -0,0 +1,805 @@
+        layout="vertical"
+        initialValues={defaultUploadInfo}
+      >
+        <Collapse
+          expandIconPosition="right"
+          accordion
+          defaultActiveKey="general"
+          css={(theme: SupersetTheme) => antdCollapseStyles(theme)}
+        >
+          <Collapse.Panel
+            header={
+              <div>
+                <h4>{t('General information')}</h4>
+                <p className="helper">
+                  {t('Upload a CSV file to a database.')}
+                </p>
+              </div>
+            }
+            key="general"
+          >
+            <Row>
+              <Col>
+                <StyledFormItem
+                  label={t('CSV File')}
+                  name="upload"
+                  required
+                  rules={[{ validator: validateUpload }]}
+                >
+                  <Upload
+                    name="modelFile"
+                    id="modelFile"
+                    data-test="model-file-input"
+                    accept=".csv"
+                    fileList={fileList}
+                    onChange={onChangeFile}
+                    onRemove={onRemoveFile}
+                    // upload is handled by hook
+                    customRequest={() => {}}
+                  >
+                    <Button aria-label={t('Select')} icon={<UploadOutlined />}>
+                      {t('Select')}
+                    </Button>
+                  </Upload>
+                </StyledFormItem>
+              </Col>
+              <Col>
+                {columns.length > 0 && (
+                  <>
+                    <Typography.Text type="success">
+                      Loaded {columns.length} column(s):
+                    </Typography.Text>
+                    {columns.map((column, index) => (
+                      <Typography.Text key={index} code type="success">
+                        {column}
+                      </Typography.Text>
+                    ))}
+                  </>
+                )}
+              </Col>
+            </Row>
+            <Divider orientation="left">Basic</Divider>
+            <Row justify="space-between">
+              <Col span={11}>
+                <StyledFormItem
+                  label={t('Database')}
+                  name="database"
+                  required
+                  rules={[{ validator: validateDatabase }]}
+                >
+                  <AsyncSelect
+                    ariaLabel={t('Select a database')}
+                    options={loadDatabaseOptions}
+                    onChange={onChangeDatabase}
+                    allowClear
+                  />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t('Select a database to upload the file to')}
+                </p>
+              </Col>
+              <Col span={11}>
+                <StyledFormItem
+                  label={t('Table Name')}
```

Comment:
@yousoph Do you know if we have guidelines on capitalization around labels? Should everything be sentence case?

Reply:
we endup capitalising all input form labels on CSS, so I don't think it matters